iXBRLViewerPlugin/viewer/src/less/inspector.less

```diff
@@ -571,18 +587,22 @@ body {
         }
 
         .powered-by {
+            .text-sm();
             text-align: center;
             position: absolute;
-            background: rgba(255,255,255,0.8);
+            //background: rgba(255,255,255,0.8);
```

Comment:
Is this still needed? If not, remove commented-out line

Reply:
Done - 8f27069